extensions/Worker.Extensions.Tables/src/TypeConverters/TableClientConverter.cs

```diff
@@ -33,6 +33,10 @@ public override ValueTask<ConversionResult> ConvertAsync(ConverterContext contex
                 {
                     return new(ConversionResult.Unhandled());
                 }
+                if (context.TargetType != typeof(TableClient))
```

Comment:
nit: does` is not` work here?
```suggestion
                
                if (context.TargetType is not typeof(TableClient))
```

second nit: space between the two if-expressions

Reply:
Please update all instances of  `if (context.TargetType != <object>)`